pennylane_lightning/core/lightning_base.py

```diff
@@ -55,7 +55,7 @@ class LightningBase(QubitDevice):
             OpenMP.
     """
 
-    pennylane_requires = ">=0.36"
+    pennylane_requires = ">=0.37"
```

Comment:
Since PL now has the dev wheels, I wonder if we can we set this to >0.37

Reply:
Should we do the same in `pyproject.toml`, `setup.py`, etc.?

Reply:
That's a good question. Maybe we can eval this after the release and investigate how well it'll work without disrupting what we have now.